src/js/components/pipelines/transforms/Inspector.jsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var React = require('react'),
+    connect = require('react-redux').connect,
+    capitalize = require('capitalize'),
+    getInVis = require('../../../util/immutable-utils').getInVis,
+    deepEquality = require('../../../util/deep-equality');
+
+function mapStateToProps(state, ownProps) {
+  var id = ownProps.dsId;
+  return {
+    transforms: getInVis(state, 'datasets.' + id + '._transforms')
+  };
+}
+
+var Inspector = React.createClass({
+  propTypes: {
+    dsId:  React.PropTypes.number,
+    transforms:  React.PropTypes.array
```

Comment:
`transforms` should be an `ImmutableList` in the store (and marked here as a `React.PropTypes.instanceof(Immutable.List)`).